subgraph/README.md

```diff
@@ -6,7 +6,7 @@ Currently there are Subgraphs deployed on the [xdai](https://thegraph.com/explor
 
 ## Queries
 
-Below there are a few ways to show how to query the Subgraph for data.  The queries show most of the information that is queryable, but there are many other filtering options that can be used, just check out the [querying api](https://thegraph.com/docs/graphql-api). These queries can be used locally or in The Graph Explorer playground.
```

Comment:
This is just a recommendation. I hope I got this right.

Reply:
It's great!